api/src/modules/auth/authentication.service.ts

```diff
@@ -81,9 +92,67 @@ export class AuthenticationService {
     };
   }
 
-  async logIn(user: User): Promise<UserWithAccessToken> {
+  private async createBackOfficeSession(
+    user: User,
+    accessToken: string,
+  ): Promise<BackOfficeSession> {
+    // We replicate what adminjs does by default using postgres as session storage (the default in memory session storage is not production ready)
+    // This implementation is not compatible with many devices per user
+    await this.backOfficeSessionRepository
+      .createQueryBuilder()
+      .delete()
+      .from(BACKOFFICE_SESSIONS_TABLE)
+      .where(`sess -> 'adminUser' ->> 'id' = :id`, { id: user.id })
+      .execute();
+
+    const currentDate = new Date();
+    const sessionExpirationDate = new Date(
+      Date.UTC(
+        currentDate.getUTCFullYear() + 1,
+        currentDate.getUTCMonth(),
+        currentDate.getUTCDate(),
+        currentDate.getUTCHours(),
+        currentDate.getUTCMinutes(),
+        currentDate.getUTCSeconds(),
+      ),
+    );
+    const backofficeSession: BackOfficeSession = {
+      sid: uid.sync(24),
```

Comment:
Any specific reason you consider pg's `gen_random_uuid()` is not safe enough crypto-wise?
According to the docs it should be, unless the instance runs out of some resource and it falls back to not-very-oosp method to generate it

If no specific reason, we might offload this [here](https://github.com/Vizzuality/tnc-blue-carbon-cost-tool/pull/130/files#diff-9745317ae908bc1323a96ac5b03927b8a548d4a7946a9a0460960b2bf7dbc09bR7), and also avoid using a lib that was last released 7 years ago